src/Kunstmaan/NodeSearchBundle/Helper/HtmlSearchCrawler.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Kunstmaan\NodeSearchBundle\Helper;
+
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Class HtmlSearchCrawler.
+ */
+final class HtmlSearchCrawler
```

Comment:
Maybe we should mark the class with the `@final` annotation and remove the final keyword? So we don't really lock the class but it should discourage users to extend from this class

Reply:
I don't really see what would be the advantage? Either it's final or not.